### PR TITLE
Clarify bug bounty payment policy

### PR DESCRIPTION
**Pull request by danpaul000**
#### Problem
Our payment policy for bug bounty rewards is not clearly documented

#### Summary of Changes
Add it

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -56,8 +56,23 @@ Eligibility:
...
+* The reference for this price is the Closing Price given by Coingecko.com on
+  that date given here:
+  https://www.coingecko.com/en/coins/solana/historical_data/usd#panel
+* For example, for all bugs submitted in March 2021, the SOL/USD price for bug
+  payouts is the Close price on 2021-03-31 of $19.49.  This applies to all bugs
+  submitted in March 2021, to be paid in mid-April 2021.
+
 Notes:
-* All locked tokens can be staked during the lockup period
+* All locked tokens can be staked during the lockup period.  Locked tokens are
```
Comment (mvines) [inline SECURITY.md:73]: I don't think we specify the lockup period anywhere in this doc yet

Comment (t-nelson) [reply SECURITY.md:73]: > I don't think we specify the lockup period anywhere in this doc yet

:100: 

I'm not sure we need to reiterate the staking eligibility of locked tokens either.  Probably enough to say something like
```
"Bug bounties are paid out in [stake accounts](link to stake accounts) with a 12 month [lockup](link to lockup)"
```
and let the links do the heavy lifting

**Review by t-nelson — APPROVED** [review]
Thanks!

File: SECURITY.md [hunk 2]
```diff
@@ -56,8 +56,22 @@ Eligibility:
 * Multiple submissions for the same class of exploit are still eligible for compensation, though may be compensated at a lower rate, however these will be assessed on a case-by-case basis
 * Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
 
-Notes:
-* All locked tokens can be staked during the lockup period
+Payment of Bug Bounties:
+* Payments for eligible bugs are distributed monthly.
+* All bug bounties submitted in a given month are paid out in the middle of the
+following month.
```
Comment (CriesofCarrots) [inline SECURITY.md:62]:
```suggestion
* Bounties for all bugs submitted in a given month are paid out in the middle of the
following month.
```

File: SECURITY.md [hunk 3]
```diff
@@ -56,8 +56,22 @@ Eligibility:
 * Multiple submissions for the same class of exploit are still eligible for compensation, though may be compensated at a lower rate, however these will be assessed on a case-by-case basis
 * Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
 
-Notes:
-* All locked tokens can be staked during the lockup period
+Payment of Bug Bounties:
+* Payments for eligible bugs are distributed monthly.
```
Comment (CriesofCarrots) [inline SECURITY.md:60]:
```suggestion
* Payments for eligible bug reports are distributed monthly.
```

**Comment by solana-grimes** [comment]
:scream: New commits were pushed while the automerge label was present.

File: SECURITY.md (hunk 2 above)
Comment (CriesofCarrots) [reply SECURITY.md]: A "bounty" is a payment. Looks like L54 could use to be fixed up as well.

**Review by CriesofCarrots — APPROVED** [review]
Thanks!

**Pull request merged**